gensim/models/word2vec.py

```diff
@@ -97,14 +97,14 @@
 
 logger = logging.getLogger(__name__)
 
+
+# Logger should be configured by this time.
 try:
     from gensim.models.word2vec_inner import train_batch_sg, train_batch_cbow
     from gensim.models.word2vec_inner import score_sentence_sg, score_sentence_cbow
     from gensim.models.word2vec_inner import FAST_VERSION, MAX_WORDS_IN_BATCH
-    logger.debug('Fast version of {0} is being used'.format(__name__))
 except ImportError:
     # failed... fall back to plain numpy (20-80x slower training than the above)
-    logger.warning('Slow version of {0} is being used'.format(__name__))
```

Comment:
Add a comment `Can't log here because logger is usually not configured at import time`